### ISSUE TITLE
Cannot load cumtomized lora weights

### ISSUE THREAD

**Issue by bestadcarry** [tloen/alpaca-lora#559]
Hello, I am completely new to the field.

I am trying to train my own lora weights by using finetune.py. I am also using the official hyperparameters. 
![image](https://github.com/tloen/alpaca-lora/assets/55209504/457460b8-d824-4f11-a6df-98b16e281e4c)

Here is the fine-tuned lora weights folder. Notice that I uploaded the gitattributes.txt by hand.
![image](https://github.com/tloen/alpaca-lora/assets/55209504/38e88916-fbcb-4564-a44a-f272cf175946)

After fine-tuning, I try to evaluate the alpaca-lora on the MMLU dataset. Here is how I load the lora module. 
![image](https://github.com/tloen/alpaca-lora/assets/55209504/873f623e-a41b-480b-b07b-119acc1881a3)

And this is the inference result on the MMLU dataset. 
![image](https://github.com/tloen/alpaca-lora/assets/55209504/5f839df3-06bd-4614-964a-5e0fdff1b120)

I find that it has exactly the same inference result with llama-7b (the base model). Below is the inference result of llama-7b. I guess I failed to load the lora module.
![image](https://github.com/tloen/alpaca-lora/assets/55209504/84cf26a1-4e71-4450-b694-974409df515b)

But when I use the same code to load the official lora weights, the lora module seems to be loaded successfully and inference generate different results.
![image](https://github.com/tloen/alpaca-lora/assets/55209504/e100ef6b-7061-4725-8b66-69e3bcd8dfef)
![image](https://github.com/tloen/alpaca-lora/assets/55209504/b3c008f4-e140-497d-9420-1a065b695e7a)

I am so confused about this and cannot figure out which part went wrong. Would someone be able to help me with this? Thank you!

The package I am using:
transformers              4.30.2                   pypi_0    pypi
peft                         0.4.0.dev0               pypi_0    pypi
python                       3.9.16               h955ad1f_3 

Thank you so much!

**Comment by JinyanSu1** [comment]
I encountered the same problem

**Comment by bestadcarry** [comment]
> I encountered the same problem

You can try to delete the below code in the finetune.py. The bin file of lora module should be at the level of the Mb size. 
```
model.state_dict = (
        lambda self, *_, **__: get_peft_model_state_dict(
            self, old_state_dict()
        )
    ).__get__(model, type(model))
```

**Comment by Yuan0320** [comment]
Hi @bestadcarry, what's your accuracy result in MMLU datasets with your own trained LoRA weights? Thanks!
